peekaboo/sample.py

```diff
@@ -470,6 +472,14 @@ def office_macros(self):
         return self.__office_macros
 
     @property
+    def office_macros_with_auto_action(self):
+        """ Determines if this sample contains any office macros which
+        automatically run at AutoOpen """
+        if not self.__office_macros_with_auto_action:
+            self.__office_macros_with_auto_action = has_office_macros_with_auto_action(self.__path)
+
+        return self.__office_macros_with_auto_action
+    @property
```

Comment:
Empty separator line missing here. The method itself is the redundant boilerplate code I meant in the previous comment. Since this method only has one user (the rule), the caching functionality is very unlikely to be used much (perhaps on ruleset re-run after deferred Cuckoo analysis - which we could and maybe should address anyway and another way, i.e. not re-running rules that already ran?)

Reply:
Out of curiosity: Did this missing line get past git-lint unnoticed?

Reply:
Out of curiosity: Did this missing line get past git-lint unnoticed?

Reply:
Style fixed with latest commits.